server/app.js

```diff
@@ -104,7 +105,8 @@ app.get('/rooms/:id/minNightlyRate', async (req, res) => {
 
 app.get('/rooms/:id/summary', async (req, res) => {
   try {
-    const response = await axios.get(`http://ec2-54-149-117-186.us-west-2.compute.amazonaws.com:5002/rooms/${req.params.id}/summary`);
+    // const response = await axios.get(`http://ec2-54-149-117-186.us-west-2.compute.amazonaws.com:5002/rooms/${req.params.id}/summary`);
+    const response = await axios.get(`http://5002/rooms/${req.params.id}/summary`);
     res.send(response.data);
   } catch (err) {
     res.send(fallback.summary);
```

Comment:
I would place a `console.log` in this catch block to see if for some reason this is throwing and error and returning fallback data to client.. Otherwise it might be something in the [client code](https://github.com/Team-Vaughan/summary-service/blob/f90520f1ee83f9fd643db33cf8e18c3fdbf84d9b/client/src/App.jsx#L34) that is sending same product id from request leaving database.